prefs.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 José Expósito <jose.exposito89@gmail.com>
+ *
+ * This file is part of gnome-shell-extension-x11gestures.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation,  either version 2 of the License,  or (at your option)  any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful,  but  WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the  GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+const { Gio, Gtk } = imports.gi;
+
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+
+function init() {}
```

Comment:
The linter can be very nitpick sometimes, so better disabled it here:

```suggestion
// eslint-disable-next-line no-unused-vars
function init() {}
```